src/backend/legendary/games.ts

```diff
@@ -751,19 +751,11 @@ class LegendaryGame extends Game {
       "Legendary's config from config.ini (before Heroic's settings):\n"
     )
 
-    try {
-      const json = JSON.parse(stdout)
-      // remove egl auth info
-      delete json['egl_parameters']
+    const json = JSON.parse(stdout)
```

Comment:
we should keep the try/catch here, if legendary fails to run the previous command for any reason it won't return a json output and `JSON.parse` fails, stopping the game launch

Reply:
I did not touched that code. Probably a merge bug.

Reply:
We still have to revert this change, even more so if this is a merge bug. This change should not happen.

Reply:
nevermind, github was showing me an outdated/cached diff